### PR TITLE
1 fix vulnerability issue

### PR DESCRIPTION
**Pull request by chetangawai**
#1 

Upgraded jquery to 3.5.1 and tested the application

### CODE REVIEW

**Comment by chetangawai** [comment]
@nish-d Please review

File: Gemfile [hunk 1]
```diff
@@ -5,3 +5,4 @@ gem "jekyll-sitemap"
 gem "jekyll-gist"
 gem 'jekyll-mentions'
 gem 'jekyll-feed'
+gem 'kramdown-parser-gfm'
```
Comment (nish-d) [inline Gemfile]: Please remove these changes.

File: Gemfile.lock [hunk 2]
```diff
@@ -0,0 +1,105 @@
+GEM
+  remote: https://rubygems.org/
```
Comment (nish-d) [inline Gemfile.lock]: Please remove these changes.

**Review by nish-d — CHANGES_REQUESTED** [review]

File: Gemfile.lock (hunk 2 above)
Comment (chetangawai) [reply Gemfile.lock]: Removed the commit

File: Gemfile (hunk 1 above)
Comment (chetangawai) [reply Gemfile]: Removed the commit @nish-d

**Review by nish-d — APPROVED** [review]
Thanks a lot @chetangawai for your PR!! Feel free to point out other issues and raise more PRs. 👍

**Pull request merged**